CHANGELOG.md

```diff
@@ -7,6 +7,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Added
+- CMS settings permission is now required when running `rewriter` queries
```

Comment:
Here you won't be adding CMS settings permission, just that you need authentication now, right?

Reply:
Yep, just fixed the message [f1fa3df](https://github.com/vtex-apps/store-indexer/pull/73/commits/f1fa3dfce4a88dabeebfcced91a33d83b5488ccf)